.circleci/config.yml

```diff
@@ -25,7 +25,8 @@ jobs:
             python -m venv venv
             . venv/bin/activate
             pip install --upgrade pip
-            pip install -e .[dev,orbital] opencv-python-headless
```

Comment:
I think removing this line is causing some of the test failures somewhere?

Reply:
Combined lines